python_basics.py

```diff
@@ -6,23 +6,24 @@
 file = "/Users/mohammadanas/Desktop/Duke MIDS/Spring 2021/SoftwareTools/Assignment_1/python-basics-biostats-821/example.txt"
 
 
-def get_data(file: list[int]) -> list[int]:
+def get_data(file: str) -> list[list[int]]:
     """This function return the data as a list of lists"""
-
     integers = []
     with open(file) as file:
         for line in file:
             l = line.split()
             integers.append(l)
         file.close()
-    return integers
+    lst1 = [int(i) for i in integers[0]]
+    lst2 = [int(i) for i in integers[1]]
+    return lst1 + lst2

```

Comment:
@sdonatileach this will give us a single list of integers right? I think for our main analyze data function, we need a list of lists to calculate the covariance and correlation so we need to make changed to this